elements/lisk-blocks/src/state_store/account_store.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { cloneDeep, isEqual, pick, uniq, uniqBy } from 'lodash';
+
+import {
+	Account,
+	StorageEntity,
+	StorageFilters,
+	StorageTransaction,
+} from '../types';
+
+type AccountWithoutAddress = Omit<Account, 'address'>;
+// tslint:disable-next-line no-null-keyword no-null-undefined-union
+type AccountKeys = Account & {
+	readonly [key: string]: string | number | boolean | null;
+};
+
+const defaultAccount: AccountWithoutAddress = {
+	publicKey: undefined,
```

Comment:
can we change this to `null`?

Reply:
I updated to consistently use undefined. Also, this was because transactions library was expecting the undefined as type